demo/src/screens/incubatorScreens/WheelPickerScreen.tsx

```diff
@@ -37,39 +37,23 @@ const useData = (initialMonth?: string, initialYear?: string, initialDays?: numb
     setShowDialog(false);
   }, []);
 
-  const onMonthChange = useCallback((item: string | undefined, _: number) => {
-    setMonth(item);
-  }, []);
-
-  const onYearChange = useCallback((item: string | undefined, _: number) => {
-    setYear(item);
-  }, []);
-
-  const onDaysChange = useCallback((item: number | undefined, _: number) => {
-    setDays(item);
+  const onMonthChange = useCallback((item: string | number, _: number) => {
+    setMonth(item as string);
   }, []);
 
-  const monthItems = useMemo(() => {
-    return _.map(months, item => ({label: item, value: item}));
+  const onYearChange = useCallback((item: string | number, _: number) => {
+    setYear(item as string);
```

Comment:
I really don't like all the `as`...
I prefer changing the `onChange` method to:
```
onChange?: ((item: string, index: number) => void) | ((item: number, index: number) => void);
```
and (for example) this method will change to:
```
const onYearChange = useCallback((item: string) => {
  setYear(item);
}, []);
```
and ignoring the internal errors (I tried to fix them, it did not seem trivial), rather than making the user casting all the time, WDYT?

Reply:
Fixed